src/MapfileStyleParser.ts

```diff
@@ -618,7 +618,35 @@ export class MapfileStyleParser implements StyleParser {
   }
 
   /**
-   * Get the GeoStyler-Style Basic Symbolizer Parameter from an Mapfile STYLE / LABEL.
+   * Get the GeoStyler-Style RasterSymbolizer from an Mapfile STYLE.
+   *
+   * @param {object} styleParameters The Mapfile Style Parameters
+   * @return {RasterSymbolizer} The GeoStyler-Style RasterSymbolizer
+   */
+  getRasterSymbolizerFromMapfileStyle(styleParameters: any): RasterSymbolizer {
```

Comment:
Use type MapfileStyle instead of object and any. This may require you to rebase onto master.